libraries/test/pom.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hva er bakgrunnen for å lage en ny modul for test? Hvilke tester tenkes kjørt der ?

Reply:
det er en steg i deling opp avhengigheter av shared. Fleste moduler bare trengte testkategorier. "shared" er jo egentlig "dataset typdefinisjoner". Så, effektivt, har jeg nå frakoblet andre tjeneste fra dataset. I neste PR er det typeendring, og da fikk jeg nesten ikke avhengighetendringer på grunn av dette

Reply:
ok